examples/#03.maxpat

```diff
@@ -562,14 +562,7 @@
 
 			}
  ],
-		"dependency_cache" : [ 			{
-				"name" : "pvm_send.maxpat",
-				"bootpath" : "~/Documents/GitHub/PVM/lib",
-				"patcherrelativepath" : "../lib",
-				"type" : "JSON",
-				"implicit" : 1
-			}
- ],
+		"dependency_cache" : [  ],
```

Comment:
this shouldn't happen - max loosing the links to the dependencies.
My educated guess is that you opened the `#03.maxpat` file directly and not through the max project file `examples.maxproj` (as described on the readme ;). This might have made the 2 `pvm_send` abstractions at the bottom red as well when you opened it - which means that they're not found.
Max projects is the thing that allows max to keep track of dependencies that are in directories other than the one where the current max patch is. In this case, the `examples.maxproj` keeps track of the directory `/PVM/lib` which holds the `pvm_send` abstraction and others.
I understand that this may sound a bit strange for ppl not used to working with Max (and even for those used to it!) but I share the explanation here so that you can know about it and keep an eye out for it in the future.

I will fetch this branch and fix it, though.

Reply:
Thank you, I am going to merge this.